src/core/init/initialize_media_source.ts

```diff
@@ -203,89 +205,97 @@ export default function InitializeOnMediaSource({
     sendingTime? : number;
   }>(1);
 
-  const loadContent$ = observableCombineLatest(
-    openMediaSource(mediaElement),
-    fetchManifest(url)
-  ).pipe(mergeMap(([ mediaSource, { manifest, sendingTime } ]) => {
+  const initiatedEME$ = emeManager$.pipe(
+    filter(({ type }) => type === "eme-init" || type === "eme-disabled"),
+    take(1)
+  );
 
-    /**
-     * Refresh the manifest on subscription.
-     * @returns {Observable}
-     */
-    function refreshManifest() : Observable<never> {
-      const refreshURL = manifest.getUrl();
-      if (!refreshURL) {
-        log.warn("Init: Cannot refresh the manifest: no url");
-        return EMPTY;
-      }
+  const loadContent$ = initiatedEME$.pipe(
```

Comment:
I don't get that one. Why waiting for this and then fetch the manifest?
Why not doing both at the same time?